content/docs/studio/get-started.md

```diff
@@ -9,13 +9,9 @@ https://www.youtube.com/watch?v=hKf4twg832g
    using your GitHub.com, GitLab.com or Bitbucket.org account, or your email
    address.
 
-   <admon type="info">
-
    When you sign up with your email address, Studio will send you a verification
    email before you can login.
 
```

Comment:
My comment was that this is standard/ expected. Probably no need to even mention it?

```suggestion
```